api/src/auth/auth.service.ts

```diff
@@ -73,7 +76,7 @@ export class AuthService {
     } else {
```

Comment:
Please remove `VALIDATION_TIMEOUT` and `MAX_VALIDATION_ATTEMPTS` from `.env.template`